plugins/cocktail-lookup.js

```diff
@@ -0,0 +1,106 @@
+const Doorman = require('../../doorman');
```

Comment:
Let's break these out into submodules, with separate repos.  Will keep the codebase smaller and easier to understand for contributors.